docs/docs/dcap.md

```diff
@@ -1,96 +1,77 @@
+system, then Conclave will use the bundled Azure DCAP client.
 
-Microsoft has written a DCAP client plugin that works with its Azure Confidential Compute virtual machines. In fact, it also works outside of Azure for single CPU systems, but this may not always be the case.
+Conclave will search the following paths for system installed DCAP Clients:
 
-Conclave's bundled Azure client plugin will *only* be used if no other plugin has been found on the system.
-The runtime will use the first `.so` it encounters in the search order below:
 ```
 /usr/lib/x86_64-linux-gnu/libdcap_quoteprov.so.1
 /usr/lib/x86_64-linux-gnu/libdcap_quoteprov.so
 /usr/lib/libdcap_quoteprov.so.1
 /usr/lib/libdcap_quoteprov.so
 ```
-Should you decide to use a bundled version (recommended), ensure the files listed above don't exist (delete or rename
-them if necessary).
 
-!!!Important
+If none of these files are present, then the bundled client will be used.
 
-    The bundled DCAP Azure client plugin works with all current generation Azure confidential VMs, as well as existing
-    CPUs. However, if Azure extends their offerings to include new platforms, or if you use a new generation system, 
-    then the bundled Azure client plugin may throw an error when attempting to load your enclave. In this case,
-    [follow the instructions below](#azure-client-plugin) to build/install an updated version of the Azure client 
-    plugin that supports the platform.
+This is the recommended way to use Conclave on Azure virtual machines, but may also be used to host enclaves outside
+the Azure cloud. It requires no additional setup.
+
+If using the Azure DCAP Client and PCCS, you may also want to set the Azure DCAP client logging level to FATAL, as the 
+default setting is quite verbose:
 
-You may want to set the Azure DCAP client logging level to FATAL as the default setting is quite verbose:
 ```sh
 export AZDCAP_DEBUG_LOG_LEVEL=FATAL
 ```
 
-If you are using the bundled version, you can skip the rest of this section. Otherwise, read on for instructions on how to manually configure the DCAP plugin.
+Other configuration options for the Azure DCAP client are documented [here](https://github.com/microsoft/Azure-DCAP-Client).
 
-#### Azure client plugin
+### Intel DCAP Client & PCCS
 
-* Identify the currently installed DCAP client plugin. It will always have one of the following names: `libdcap_quoteprov.so.1` or `libdcap_quoteprov.so`. You might find other similarly named files, but they won't be used as a plugin.
-```sh
-ls /usr/lib/x86_64-linux-gnu/libdcap_quoteprov.so*
-ls /usr/lib/libdcap_quoteprov.so*
-```
+Intel provides reference DCAP client and PCCS implementations. If you do not want to rely on externally hosted services
+for attestation, you can set up your own PCCS based on the Intel implementation. This requires that you create an
+account with Intel and generate an API key so that your PCCS can communicate with the Intel provisioning servers.
```

Comment:
The following points in the current [published version](https://docs.conclave.net/dcap.html) appear to be missing:

1. Intel DCAP client is a good option if users want to run their code on bare-metal machines.
2. DCAP client package installation instruction for Ubuntu as an example: sudo apt-get install libsgx-dcap-default-qpl